packages/eslint-plugin/docs/rules/restrict-plus-operands.md

```diff
@@ -18,70 +18,145 @@ This rule reports when a `+` operation combines two values of different types, o
 ### ❌ Incorrect
 
 ```ts
-var foo = '5.5' + 5;
-var foo = 1n + 1;
+let foo = '5.5' + 5;
+let foo = 1n + 1;
 ```
 
 ### ✅ Correct
 
 ```ts
-var foo = parseInt('5.5', 10) + 10;
-var foo = 1n + 1n;
+let foo = parseInt('5.5', 10) + 10;
+let foo = 1n + 1n;
 ```
 
 ## Options
 
-### `checkCompoundAssignments`
+### `allowAny`
```

Comment:
Per https://github.com/typescript-eslint/typescript-eslint/issues/6110#issuecomment-1481954671 - these new areas of options should have very explicit caveats & suggestions not to use them.